apps/finance/contracts/Finance.sol

```diff
@@ -26,11 +26,11 @@ contract Finance is EtherTokenConstant, IsContract, AragonApp {
     bytes32 public constant DISABLE_PAYMENTS_ROLE = keccak256("DISABLE_PAYMENTS_ROLE");
 
     uint64 public constant MAX_PAYMENTS_PER_TX = 20;
+    uint64 internal constant NO_PAYMENT = 0;
```

Comment:
Good point, slipped my mind! I'll get on that!

Reply:
Although, I just realized, these are internal fields, does that still apply? 

I did some testing with 
``` 
pragma solidity ^0.4.0; 

contract Test { 
    uint64 constant internal thing = 43; 
    uint64 constant internal thing2 = 43;
    uint64 constant internal thing3 = 43; 
    uint64 constant internal thing4 = 43; 
} 
``` 
So it turns out changing the constants uint type has an extremely minimal effect on the gas, so because of this it's fine to make them `uint256`. 
However, the real culprit is the `public` constant, why does this need to be public? I don't see it being used anywhere. It's inclusion into the contract alone costs about 50K extra gas on deploy.

Reply:
With internal fields the impact is not that bad, I was actually testing the cost of using the getter generated by the `public constant`. I think `MAX_PAYMENTS_PER_TX` could be made internal as it is not used by the tests.

@sohkai what do you think? You made the majority of constants internal in #453 but not this one, was there a reason?

Reply:
I thought it'd be useful information to be able to query for, since it's a configuration parameter that may change in the future.

I'm not opposed to making it internal though.

Reply:
If it were to be changed later we should remove `constant` and make a setter function for the field then no?

Reply:
Ah, I meant across different versions of the app. Maybe later on we decide in a new version of the contract to use `50` or something. But realistically there's very little a contract or client could do even knowing about this parameter, and it's something really easy to expose later.